test/app-luatest/httpd.py

```diff
@@ -80,6 +80,8 @@ def post_handle(env, response):
     for key,value in iter(env.items()):
         if "HTTP_" in key:
             headers.append((key[5:].lower(), value))
+    if env.get("CONTENT_TYPE", None) != None:
```

Comment:
Nit: `if env.get("CONTENT_TYPE"):` is simpler and should work (at least it works for usual `dict`).